arches/app/views/api.py

```diff
@@ -1095,8 +1146,28 @@ def get(self, request, resourceid):
         perm = "read_nodegroup"
 
         resource = Resource.objects.get(pk=resourceid)
-        graph = Graph.objects.get(graphid=resource.graph_id)
+        published_graph = models.PublishedGraph.objects.get(publication=resource.graph_publication, language=translation.get_language())
```

Comment:
Found that an import went missing here (assuming after tons of merges). Reproduced by clicking "View Report" button from Edit Log activity:

```py
2024-04-02 16:52:28   File "/web_root/arches/arches/app/views/api.py", line 1149, in get
2024-04-02 16:52:28     published_graph = models.PublishedGraph.objects.get(publication=resource.graph_publication, language=translation.get_language())
2024-04-02 16:52:28 NameError: name 'translation' is not defined
```